biz.aQute.bndlib/src/aQute/bnd/build/Project.java

```diff
@@ -2729,13 +2763,14 @@ public Jar pack(String profile) throws Exception {
 	 */
 
 	public void baseline() throws Exception {
-		ProjectBuilder b = getBuilder(null);
-		for (Builder pb : b.getSubBuilders()) {
-			ProjectBuilder ppb = (ProjectBuilder) pb;
-			Jar build = ppb.build();
-			getInfo(ppb);
+		try (ProjectBuilder b = getBuilder(null);) {
+			for (Builder pb : b.getSubBuilders()) {
+				ProjectBuilder ppb = (ProjectBuilder) pb;
+				Jar build = ppb.build();
```

Comment:
The jar is associated with the builder that returns it. So when the builder is closed, the jar will also be closed. The builder will be closed when the project builder is closed.